### PR TITLE
Icebox library ruin better

### PR DESCRIPTION
**Pull request by EuSouAFazer**
## About The Pull Request

Makes icebox's library ruin better

## Why It's Good For The Game

it looks better and has slightly more random paperwork. not much to say.

## Changelog

-> the regular plating has been changed for iced over plating
-> added a bunch of random burocracy spawner to fit with the theme
-> added a book binder

:cl:
qol: Icebox's Library ruin now looks slightly better
/:cl:

### CODE REVIEW

**Review by MMMiracles — CHANGES_REQUESTED** [review]
the wood and carpet floors should be a subtype with the icemoon atmos set instead of being var-edited. i know you didn't do this change but since you're touching it you might as well fix that issue.

rest looks fine, just need the floor changes

**Comment by EuSouAFazer** [comment]
> the wood and carpet floors should be a subtype with the icemoon atmos set instead of being var-edited. i know you didn't do this change but since you're touching it you might as well fix that issue.
> 
> rest looks fine, just need the floor changes

wow I didn't even notice that, that's fucked up tbh. When I get home in a bit I'll fix it dworry.

File: _maps/RandomRuins/IceRuins/icemoon_underground_library.dmm [hunk 1]
```diff
@@ -399,6 +427,12 @@
 	initial_gas_mix = "ICEMOON_ATMOS"
 	},
 /area/ruin/unpowered/buried_library)
+"EE" = (
+/obj/effect/spawner/random/bureaucracy/paper,
+/turf/open/floor/wood{
+	initial_gas_mix = "ICEMOON_ATMOS"
+	},
```
Comment (JohnFulpWillard) [inline _maps/RandomRuins/IceRuins/icemoon_underground_library.dmm:434]: Make this a subtype in code instead please
This shouldn't even work

**Comment by JohnFulpWillard** [comment]
oh mmmiracles already pointed it out
damn mappers

**Pull request closed without merging**